src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -146,6 +146,7 @@ private PackageSourceMoniker SelectedSource
             solutionManager.NuGetProjectUpdated += SolutionManager_ProjectsUpdated;
             solutionManager.NuGetProjectRenamed += SolutionManager_ProjectRenamed;
             solutionManager.ActionsExecuted += SolutionManager_ActionsExecuted;
+            solutionManager.AfterNuGetCacheUpdated += SolutionManager_CacheUpdated;
```

Comment:
I think instead of creating a new SolutionManager_CacheUpdated function, SolutionManager_ActionsExecuted function can be reused.